phpstan.neon

```diff
@@ -4,7 +4,6 @@ includes:
 parameters:
     level: 4
     ignoreErrors:
-        - '#Access to an undefined property DOMNamedNodeMap::\$length#'
```

Comment:
PHPStan reported that this error did not occur anymore, so I have removed the ignored error.